### PR TITLE
Feat#24:Developed card UI

### PR DESCRIPTION
**Pull request by pragyamishra56**
_For an example of how to fill this template out, [see this Pull Request in the Wiki](https://github.com/YurisCodingClub/DevCV/wiki/How-to-create-a-pull-request)._

## Description

No changes made; only added the folder for the page displayed in the screenshot.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|✓| :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### Before
Screenshot 2023-10-26 141349

### After

Screenshot 2023-10-26 140645

### CODE REVIEW

File: public/Git_Card_UI/index.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
```
Comment (YuriDevAT) [inline public/Git_Card_UI/index.html:1]: This has to be a React Component (which means HTML within a JavaScript Component, JSX), like all the other components as well.

File: public/Git_Card_UI/style.css [hunk 2]
```diff
@@ -0,0 +1,276 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@200;300;400;500;600;700&display=swap");
```
Comment (YuriDevAT) [inline public/Git_Card_UI/style.css:1]: As mentioned a few times, the project is using TailwindCSS, not vanilla CSS.

**Review by YuriDevAT — COMMENTED** [review]
Thank you for your contribution 🚀 .

Unfortunately, there was miscommunication throughout the issue discussion. This is a React project, which means you have to follow common syntax like HTML inside a component, add TailwindCSS directly inside the component (inline styles), name the component "Card.tsx" and move it inside the components folder inside the src folder, which contains all the other components as well.

I am afraid that this PR does not make it in time for Hacktoberfest, and I am sorry. Since the issue was not labeled as good first issue, I assumed you are familiar with the code base and your questions refer purely to the design implementation 😔

**Comment by pragyamishra56** [comment]
> Thank you for your contribution 🚀 .
> 
> Unfortunately, there was miscommunication throughout the issue discussion. This is a React project, which means you have to follow common syntax like HTML inside a component, add TailwindCSS directly inside the component (inline styles), name the component "Card.tsx" and move it inside the components folder inside the src folder, which contains all the other components as well.
> 
> I am afraid that this PR does not make it in time for Hacktoberfest, and I am sorry. Since the issue was not labeled as good first issue, I assumed you are familiar with the code base and your questions refer purely to the design implementation 😔

Thank you for your understanding and guidance. I apologize for any miscommunication. I'll ensure to follow the React project's conventions for future contributions. Although it won't make it for Hacktoberfest, I appreciate the opportunity to learn and contribute to the project. 😊

Is there a possibility to try again with your guidance on this issue, if you're willing to provide another chance?

**Comment by YuriDevAT** [comment]
For sure. When Hacktoberfest is finished I will update the issues to make them more clear (and / or) offer help. Also, I do have two projects which are only HTML/CSS and JS, if this is something you are more comfortable will. New issues will also be added there in the upcoming days

HTML Quiz: https://github.com/YurisCodingClub/html-tag-quiz
RegexIt: https://github.com/YurisCodingClub/regex-it

**Pull request closed without merging**